integreat_cms/cms/utils/linkcheck_utils.py

```diff
@@ -252,36 +252,33 @@ def replace_links(
         region_msg,
         user_msg,
     )
-    models = [PageTranslation, EventTranslation, POITranslation, ImprintPageTranslation]
     with update_lock:
-        for model in models:
-            filters = {}
-            if region:
-                filters[f"{model.foreign_field()}__region"] = region
-
-            for translation in model.objects.filter(**filters).distinct(
-                model.foreign_field(), "language"
-            ):
-                new_translation = deepcopy(translation)
-                for link in translation.links.select_related("url"):
+        links = get_region_links(region) if region else Link.objects.all()
```

Comment:
Because `replace_links` is used in the management command too and it is possible to leave "region" blank there, `Link.objects.all()` was added for the case of `region=None`

Reply:
Could you please write a comment in the code for that so we don't forget? :)

Reply:
Good idea :) done